src/dmd/cppmanglewin.d

```diff
@@ -1036,7 +1036,9 @@ extern(D):
             mangleName(p, dont_use_back_reference);
             // Mangle our string namespaces as well
             for (auto ns = p.cppnamespace; ns !is null; ns = ns.cppnamespace)
-                mangleName(ns, dont_use_back_reference);
+                if (ns.ident !is null)
+                    mangleName(ns, dont_use_back_reference);
+
```

Comment:
Can there be any `null` component ? I guess not, so this probably should be on the `for` condition (`ns !is null && ns.ident !is null`).

Reply:
`extern(C++, "ns", "",  "a")` would produce that.

Reply:
Changed